packages/liquidator/src/proxyTransactionWrapper.js

```diff
@@ -74,8 +74,9 @@ class ProxyTransactionWrapper {
           return this.web3.utils.isAddress(x) || x === "";
         },
       },
-      maxReserverTokenSpent: {
-        value: MAX_UINT_VAL,
+      maxAcceptableSlippage: {
+        // default to reverting on >10% slippage on trades.
+        value: this.toWei("0.1"),
         isValid: (x) => {
           return typeof x == "string";
```

Comment:
Why not enforce that this is between 0 and 100% at least? or do you think thats nonsensical

Reply:
na that seems reasnoble. I've added it.